src/components/pages/Step5.vue

```diff
@@ -13,8 +13,8 @@
                 <v-btn v-show="show_controls" :disabled="busy || invalid" @click="changeState">
                     {{ buttonCaption }}
                 </v-btn>
-                <p v-if="state === 'recording'" style="color: orange;">Videos are recording, do not refresh</p>
-                <p v-if="state === 'processing'" style="color: green;">Videos are uploading, do not refresh</p>
+                <p v-if="state === 'recording'">Videos are recording, do not refresh</p>
+                <p v-if="state === 'processing'">Videos are uploading, do not refresh</p>
```

Comment:
@AlbertoCasasOrtiz maybe I am missing something but how do we distinguish between uploading and processing? My understanding is that it is the same state `processing`. We somehow can make a distinction, since the button to start recording is inactive when it is uploading, but active again when upload is done. Maybe your PR is all good, I am just curious.